server/auth/gitlab/gitlab.go

```diff
@@ -19,14 +24,98 @@ const (
 // Gitlab account.
 type Config struct {
 	config    *oauth2.Config
-	baseurl   string
 	group     string
 	whitelist map[string]bool
 	allusers  bool
+	apiurl    string
```

Comment:
The README needs to be updated with these changes.

Reply:
Done